packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.component.tsx

```diff
@@ -110,12 +116,13 @@ const PatientBanner: React.FC<PatientBannerProps> = ({
         </ConfigurableLink>
         <div className={styles.buttonCol}>
           {!hideActionsOverflow && (
-            <div ref={overFlowMenuRef}>
+            <div className={styles.overflowMenuContainer} ref={overflowMenuRef}>
               <CustomOverflowMenuComponent
+                deceased={isDeceased}
                 menuTitle={
                   <>
                     <span className={styles.actionsButtonText}>{t('actions', 'Actions')}</span>{' '}
-                    <OverflowMenuVertical size={16} style={{ marginLeft: '0.5rem' }} />
+                    <OverflowMenuVertical size={16} style={{ marginLeft: '0.5rem', fill: '#78A9FF' }} />
```

Comment:
Move all this to SCSS. Also, is it possible to use a standard color variable here?